packages/dds/legacy-dds/api-report/legacy-dds.legacy.alpha.api.md

```diff
@@ -1,14 +1,9 @@
-## Alpha API Report File for "@fluid-private/legacy-dds"
+## Alpha API Report File for "@fluidframework/legacy-dds"
 
 > Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
 
 ```ts
 
-// @alpha @legacy (undocumented)
-export type FullyReadonly<T> = {
```

Comment:
have you looked at our partner code to ensure removing this won't be a problem?

Reply:
I'll make sure this change works in loop repo first using their own packages. I'll follow up in another PR if that's the case.